contents/handbook/people/hiring-process.md

```diff
@@ -94,31 +92,11 @@ This is to allow both PostHog and the candidate to assess whether the candidate
 
 Candidates who are unsuccessful at this stage should receive a personalized email with feedback. 
 
-### Interview 2 - Technical
+### Interview 2
 
-The second step is the technical interview. This is usually 2 PostHog team members spending an hour with the candidate (at the same time).
+For engineering, this is a 2-part interview. The first part is a half hour interview with Tim about your previous experiences and what motivates you to come join us. The second part is an hour long technical interview with an engineer. This might be architecture design or diving more into past technical experiences.
```

Comment:
Should be definitive on either architecture design or diving more into past technical experiences/saying we'll do both as they're quite different expectations to prepare for